qa/common/check_copyright.py

```diff
@@ -36,7 +36,7 @@
              'jmx', 'gz', 'json', 'pdf', 'so', 'onnx')
 REPO_PATH_FROM_THIS_FILE = '../..'
 SKIP_PATHS = (
-    'builddir', 'build/libevhtp', 'build/onnxruntime',
+    'builddir', 'build/libdcgm', 'build/libevhtp', 'build/onnxruntime',
```

Comment:
@dzier Thanks for fixing this. Can you also fix the copyright line for metrics.cc in this PR? Just remove the '(c)' after Copyright since it is causing the copyright to be split into 2 lines because of the formatter.